features/jest/client/src/main/java/org/opennms/features/jest/client/bulk/BulkRequest.java

```diff
@@ -51,12 +52,14 @@
     private final int retryCount;
     private int retries = 0;
     private BulkWrapper bulkAction;
+    private final ClientWithCircuitBreaker clientWithCircuitBreaker;
 
     public BulkRequest(final JestClient client, final List<T> documents, final Function<List<T>, BulkWrapper> documentToBulkTransformer, int retryCount) {
         this.client = Objects.requireNonNull(client);
         this.transformer = Objects.requireNonNull(documentToBulkTransformer);
         this.documents = new ArrayList<>(Objects.requireNonNull(documents));
         this.retryCount = retryCount;
+        this.clientWithCircuitBreaker = client instanceof ClientWithCircuitBreaker? (ClientWithCircuitBreaker) client : null;
```

Comment:
Why can we not alter the signature of the constructor? Or as an alternative, always wrap the connection?